crates/bioauth-flow-rpc/src/lib.rs

```diff
@@ -366,14 +369,42 @@ where
             )
             .map_err(AuthenticateError::RuntimeApi).map_err(errtype)?;
 
```

Comment:
Let's add a log event here to indicate that we're sending the tx

Reply:
https://github.com/humanode-network/humanode/pull/1018/commits/ee39f3df28b925059803d408e07de5690ef1c1a3